integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/steps/ProcessorSteps.java

```diff
@@ -210,7 +211,8 @@ public void processorOfBridgeIsNotExistingWithinMinutes(String processorName, St
                 .conditionEvaluationListener(new AwaitilityOnTimeOutHandler(
                         () -> ProcessorResource
                                 .getProcessorResponse(context.getManagerToken(), bridgeContext.getId(),
-                                        processorId)))
+                                        processorId)
+                                .then().log().all()))
```

Comment:
@sutaakar  is `all` referring to all items of a collection or all levels of severity?

Reply:
It means to log a complete response, with headers and such